accepted/0000-install-peer-deps.md

```diff
@@ -0,0 +1,144 @@
+# Install Peer Dependencies
+
+## Summary
+
+Install `peerDependencies` along with packages that peer-depend on them.
+
+Ensure that a validly matching peer dependency is found at or above the
+peer-dependant's location in the `node_modules` tree.
+
+## Motivation
+
+Due to some of the difficulties that `peerDependencies` present with the
+installer as of npm v6, `peerDependencies` are not installed by default
```

Comment:
wasn't this as of npm 3, not 6?

Reply:
I believe so, but technically both are true ;)  I should track down when this actually started, I guess.